its/ruling/src/test/resources/eclipse-jetty/java-S6212.json

```diff
@@ -71,6 +71,10 @@
 590,
 592,
 599,
+842,
```

Comment:
This looks like an FP because `i` is used:
```
for (int i = 0; i < __preprepared.length; i++)
```

Reply:
I guess the issue here is with S6212 that suggests using `var` on the local variable, even though the PR does not touch that rule

Reply:
My Bad, I confused S6212 result with S1481. It's not a S1481 false-positive.